cli_tools/gce_vm_image_import/cli/cli.go

```diff
@@ -89,6 +92,15 @@ func Main(args []string) error {
 	return nil
 }
 
+func validateDirectoryAndMakeAbsolute(workflowRoot string) string {
+	abs, err := filepath.Abs(workflowRoot)
+	if err != nil {
+		panic(err)
+	}
+	assert.DirectoryExists(abs)
+	return abs
+}
+
```

Comment:
This seems like a generic function. Some comments:
* workflowRoot is too specific from the context of this func, it can be e.g. `directory`
* This would fit in a FileSystemUtils mentioned in a comment regarding assert.DirectoryExists
* This func has two responsibilities, consider splitting validation and conversion to absolute path. Otherwise, consider giving it a more descriptive name, e.g. `validateDirectoryExistsAndConvertPathToAbsolute` (even a bigger reason to split in two funcs)

Reply:
Makes sense -- moved to `cli_tools/common/utils/files/files.go`